webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/NodeConfigData.scala

```diff
@@ -960,14 +959,16 @@ class TestNodeConfiguration(
   val commonTechnique:                                                      Technique                  =
     techniqueRepository.unsafeGet(TechniqueId(TechniqueName("common"), TechniqueVersionHelper("1.0")))
   def commonVariables(nodeId: NodeId, allNodeInfos: Map[NodeId, NodeInfo]): Map[ComponentId, Variable] = {
```

Comment:
most of changes here look like that: I needed to add the component id as a key to be consistent with the new `technique.getAllVariableSpecs` method.